src/view/pointPath.js

```diff
@@ -1,29 +1,44 @@
-export const createPointPathTemplate = () => (
-  `<li class="trip-events__item">
+import dayjs from 'dayjs';
+
+export const createPointPathTemplate = (point) => {
```

Comment:
используй деструктуризацию в параметрах
```suggestion
export const createPointPathTemplate = ({offers, type, basePrice, destination}) => {
```